hw/ip/rv_dm/doc/_index.md

```diff
@@ -30,6 +30,23 @@ The debug system is compliant with the [RISC-V Debug Specification 0.13.2](https
 
 # Theory of Operations
 
+## Memory Maps
+
+### TL-UL device
+The memory map accessible over the TL-UL device interface is documented in the [Debug Memory](https://github.com/lowRISC/opentitan/blob/master/hw/vendor/pulp_riscv_dbg/doc/debug-system.md#debug-memory) section of the [PULP RISC-V Debug System Documentation](https://github.com/lowRISC/opentitan/blob/master/hw/vendor/pulp_riscv_dbg/doc/debug-system.md).
+Note this contains a mixture of read only and read-write regions and which is which isn't documented.
+The read-write regions are:
+
+ - ``0x100`` - ``0x10c``: Halted, Going, Resuming, Exception
```

Comment:
@sriyerg one thing we could do (to ease testing) is to place some custom logic downstream of `tlul_adapter_sram` to error back when writes happen out of the space @GregAC has specified.  I'm not personally a huge fan of this since the designs could become mismatched, but in the short term, it could save DV having to do a bunch of manual work. 

Or would it be better if we split into multiple windows?

Reply:
All we need for the tests to work is a RAL model that accurately represents this region. We either do it in HJSON or I can manually write the SV directly (I am assuming it's just a handful of CSRs / ROM / buffers). 

The HJSON will be more readable and easy to make changes to later if needed, so I prefer that. We can make this DV-only HJSON that is not used for RTL generation (so we do not need to touch `data/rv_dm.hjson`). I hope `reggen` does not have other assumptions that prevents this type of usage.